src/main/java/emissary/pickup/PickUpPlace.java

```diff
@@ -552,6 +563,12 @@ protected boolean processDataObject(IBaseDataObject d, String fixedName, File th
         dataObjectCreated(d, theFile);
         logger.info("**Deploying an agent for {} and object {} forms={} simple={}", fixedName, d.getInternalId(), d.getAllCurrentForms(),
                 (simpleMode ? "simple" : ""));
+
+        // If object tracing log that agent is being deployed for fixedName (filename)
```

Comment:
Are we sure this is where the PickUp event should be emitted?  For input files containing multiple sessions, this will emit multiple PickUp events that vary only by timestamp unless we're also emitting additional IBDO fields.

Reply:
My understanding of the requirements is that we needed a message or trace at the object level. Other attributes will be added to the message by overriding the `emitLifecycleEvent()` method. If that makes sense?